jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextHandler.java

```diff
@@ -1197,7 +1197,7 @@ protected boolean handleByContextHandler(String pathInContext, ContextRequest re
             return true;
         }
 
-        return super.handleByContextHandler(pathInContext, request, response, callback);
+        return false;
```

Comment:
I don't like this always false return.

I think this method would be better written like:
```java
    protected boolean handleByContextHandler(String pathInContext, ContextRequest request, Response response, Callback callback)
    {
        if (!request.instanceof ServletContextRequest && isProtectedTarget(pathInContext))
            return false;
        return super.handleByContextHandler(pathInContext, request, response, callback);
    }
```

Reply:
I see what you mean, but this code is incorrect with `isProtectedTarget(pathInContext)` here.

I think this is correct:
```java
boolean initialDispatch = request instanceof ServletContextRequest;
if (!initialDispatch)
    return false;
return super.handleByContextHandler(pathInContext, request, response, callback);
```